pallets/automation-price/src/lib.rs

```diff
@@ -1115,9 +1132,9 @@ pub mod pallet {
 				//
 				let price_matched_target_condtion =
 					if task.trigger_function == TRIGGER_FUNC_GT.to_vec() {
-						task.trigger_params[0] < this_task_asset_price.amount
+						task.trigger_params[0] < this_task_asset_price.value
```

Comment:
so if the trigger_function is "GT", meaning the actual value is greater than the set target. 

The logic has no proble, but writing this line like 
```
if GT
    this_task_asset_price.value > task.trigger_params[0]
``` 
would be more intuitive ...